extensions/integration-tests/src/testConfig.ts

```diff
@@ -83,6 +93,11 @@ export async function getDefaultTestingServer(): Promise<TestServerProfile> {
 	return servers[0];
 }
 
+export async function getBdcServer(): Promise<TestServerProfile> {
+	let servers = await getTestingServers();
+	return servers.filter(s => s.version === '2019')[0];
```

Comment:
we need to handle the scenario when running the integration test locally, raise an error with proper message if the environment variables are not set.

@kburtram , in this case, do you think we should skip the test or error out? personally I think we should error out to avoid losing test coverage.

Reply:
it seems you haven't resolved this comment

Reply:
@alanrenmsft was planned to add script to set it in next PR